engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -40,15 +40,15 @@
     @Replicate
     public float lightDiffuseIntensity = 1.0f;
     @Replicate
-    public float lightSpecularIntensity = 0.1f;
+    public float lightSpecularIntensity = 0.03f;
```

Comment:
Make torches much less shiny,  yet a little more than sunlight.  You can definitely notice it when you drop a torch right in front of a block,  but it is much more subtle.

![terasology-160413203905-1280x720](https://cloud.githubusercontent.com/assets/5922109/14516549/fa40d2b4-01b8-11e6-8b60-2f87ebb0e61f.jpg)